pkg/apps/deployer/deployer_controller_test.go

```diff
@@ -1182,6 +1182,11 @@ func TestMakeDeployerPod(t *testing.T) {
 				p.Spec.SecurityContext = nil
 				p.Spec.ReadinessGates = nil
 				p.Spec.RuntimeClassName = nil
+				// k8s 1.16
+				p.Spec.PreemptionPolicy = nil
+				p.Spec.Overhead = nil
+				p.Spec.EphemeralContainers = nil
+				p.Spec.TopologySpreadConstraints = nil
```

Comment:
@tnozicka ptal

Reply:
this change lgtm, would be nice to clean up the vendoring commits though